src/plugins/PluginFactory.cpp

```diff
@@ -88,6 +89,7 @@ void PluginFactory<PluginBase>::registerDefaultPlugins(){
     REGISTER_PLUGIN(ResultWriter);
     REGISTER_PLUGIN(SysMonOutput);
     REGISTER_PLUGIN(TestPlugin);
+	REGISTER_PLUGIN(TimerWriter);

```

Comment:
indentation

Reply:
well everything else was wrongly indented ...

Reply:
we should get clang-format for ls1 :D

Reply:
there is a clang-format file, but we don't enforce it because of historic reasons:
some people don't want to reformat/touch everything, as that messes up the history...